src/components/InfoTodo/InfoTodo.tsx

```diff
@@ -0,0 +1,89 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { updateTodo, deleteTodo } from '../../api/todos';
+
+type Props = {
+  todo: Todo;
+  handleLoadTodos: () => void;
+  isEditing?: boolean;
+};
+
+export const InfoTodo: React.FC<Props> = ({
+  todo,
+  handleLoadTodos,
+  isEditing,
+}) => {
+  const { title, completed } = todo;
+  const [isLoading, setIsLoading] = useState(false);
+  const [value, setValue] = useState(title);
+  const [isDoubleClick, setIsDoubleClick] = useState(false);
```

Comment:
let`s create a more meaningful state name because it hard to understand for what it created

Reply:
agree with @Dikey945